src/main/java/net/imagej/plugins/io/mesh/stl/BinarySTLFormat.java

```diff
@@ -30,13 +30,15 @@
 
 package net.imagej.plugins.io.mesh.stl;
 
-import com.sun.javafx.geom.Vec3f;
+import net.imagej.plugins.io.mesh.Vertex;
 
 import java.nio.ByteBuffer;
 import java.nio.ByteOrder;
 import java.util.ArrayList;
 import java.util.List;
 
+import org.apache.commons.math3.geometry.euclidean.threed.Vector3D;
```

Comment:
Actually, is this change directly related to the change over to `Mesh`? Maybe this should be yet another separate commit? From JavaFX's `Vec3f` to Commons Math 3's `Vector3D`?